Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -123,13 +110,15 @@ CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedStrin
     }
 
     // Get the default locale for this user.
-    std::wstring localeName = _GetUserDefaultLocaleName();
+    // If the default locale is returned, find that locale name, otherwise use "en-us".
+    wchar_t localeNameBuf[LOCALE_NAME_MAX_LENGTH];
+    int defaultLocaleSuccess = GetUserDefaultLocaleName(localeNameBuf, LOCALE_NAME_MAX_LENGTH);
+    const wchar_t* localeName = defaultLocaleSuccess ? localeNameBuf : c_defaultUserLanguage;
 
```

Comment:
could be factored into a private method. This could be a static as well, if we don't concern ourselves with locale changing when app is running.